Tasks/ServiceFabricDeploy/task.json

```diff
@@ -0,0 +1,96 @@
+{
+  "id": "c6650aa0-185b-11e6-a47d-df93e7a34c64",
+  "name": "ServiceFabricDeploy",
+  "friendlyName": "Service Fabric Application Deployment",
+  "description": "Deploy a Service Fabric application to a cluster.",
+  "author": "Microsoft Corporation",
+  "helpMarkDown": "",
```

Comment:
Our tasks typically have an fwlink to the documentation. It would be good for you to check with @chrisrpatterson if he has some recommendation for you here. I'm not sure what he has in mind.